_data/notebooks_and_apps.json

```diff
@@ -0,0 +1,176 @@
+{
+  "@context": "https://schema.org",
+  "@type": "ItemList",
+  "name": "reference notebooks and apps",
+  "itemListElement": [
+    {
+      "@type": "WebPage",
```

Comment:
What is the object this is pointing at? Is it the study itself? A representation of the associated analytical tools?

Reply:
It points to the study itself which has been used for either showing the usage of a tool or an exploration of the finding in the paper associated to the study